test/test.js

```diff
@@ -1517,6 +1519,78 @@ test('connection state change does not fire "connectionstatechange" in closed Tr
 	sendTransport.removeAllListeners('connectionstatechange');
 }, 500);
 
+test('RemoteSdp properly handles multiple streams of the same type in planB', async () =>
+{
+	let remoteSdp = new RemoteSdp({ planB: true });
+	let sdp = undefined;
+	let sdpObject = undefined;
+
+	await remoteSdp.receive({
```

Comment:
Is this passing ESLint? `{` should be in next line.

Reply:
yes, it was passing. Updated.